scripts/beaker/mitchish70.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CONFIG_PATH=configs/mitchish70-s3.yaml
+NUM_NODES=4
+SEED=3423
+INIT=fan_in
+RUN_NAME="fan-in-init-${SEED}"
+ARGS="--run_name=${RUN_NAME} --data.seed=6198 --seed=${SEED} --model.init_fn=${INIT} --model.init_std=0.006 --model.init_cutoff_factor=3 --device_train_microbatch_size=4 --model.flash_attention=true --fused_loss=true --evaluators=[] --stop_at=500 --wandb.group=mitchish70-ablate-init --save_interval_ephemeral=100"
```

Comment:
Looks like this isn't the final config anyways.

Reply:
Cleaned up in 92d2a08c.